core/src/serde.rs

```diff
@@ -0,0 +1,39 @@
+use proc_macro2::Span;
```

Comment:
I'd prefer that this impl be colocated with `IdentString`, rather than having a root module for serde functionality.

Reply:
Alright, I can do that.